static/monorail-2.0.yaml

```diff
@@ -617,6 +625,271 @@ paths:
       - filesRead
       x-view: file.2.0.json
     x-swagger-router-controller: files
+  /ibms:
+    get:
+      description: 'Get a list of all In Band Management settings that have been associated
+
+        with nodes. 
```

Comment:
@yyscamper Because this patch has been sitting in review for ~14 days, it needs to be rebased with current master, that got changed since then! This will go away after rebasing.